autoapi/mappers/python.py

```diff
@@ -267,6 +275,17 @@ class PythonPackage(PythonPythonMapper):
 class PythonClass(PythonPythonMapper):
     type = 'class'
 
+    @PythonPythonMapper.args.getter
+    def args(self):
+        if self._args:
+            return self._args
+
+        for child in self.children:
+            if child.short_name == '__init__':
+                return child.args
```

Comment:
If this is only for the case where this object is a class type, perhaps this should be more explicit about the fact, instead of just looking for `__init__`.

Reply:
I don't understand what you mean.

Reply:
The case above is implying that any object with a child `__init__` can be called with the arguments of the `__init__`.

What it should say is if the object type is a class, and has a child `__init__`, the object can be called with arguments of the `__init__`.

Reply:
This property override exists on the `PythonClass` mapper so it applies to classes only. `PythonPythonMapper` uses `self._args` only.

Reply:
Roger. This is plenty then